src/app/core/components/common/filters/filters.component.html

```diff
@@ -49,44 +54,59 @@ <h3 class="govuk-heading-s govuk-!-margin-bottom-0">{{ selected.displayName }}</
         Apply filters
       </button>
 
-      @for (filter of filters; track filter.name) {
+      @for (filter of filters; track filter.name; let index = $index; let last = $last) {
         <div class="govuk-form-group">
           <fieldset class="govuk-fieldset">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">{{ filter.displayName }}</legend>
-
-            @if (filter.search) {
-              <input
-                class="govuk-input search-filter"
-                id="search-filter"
-                [name]="filter.name + 'search-filter'"
-                type="text"
-                [placeholder]="'Search for ' + filter.displayName.toLowerCase()"
-                [(ngModel)]="searchTerms[filter.name]"
-              />
-            }
-            @if (getFilteredValues(filter, searchTerms[filter.name]); as filteredValues) {
-              @for (value of filteredValues; track $index) {
-                <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes">
-                  <div class="govuk-checkboxes__item">
-                    <input
-                      #checkboxes
-                      class="govuk-checkboxes__input"
-                      [id]="'type-value-' + $index"
-                      [name]="filter.name"
-                      type="checkbox"
-                      [attr.aria-label]="'Select checkbox value ' + value"
-                      [value]="value"
-                      (change)="onCheckChanged(filter, value, $event)"
-                      [checked]="isChecked(filter.name, value)"
-                    />
-                    <label class="govuk-label govuk-checkboxes__label" for="type"> {{ value }} </label>
-                  </div>
-                </div>
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s" [class.collapsed]="!filter.isCollapsed">
+              <div class="moj-pagination__item moj-pagination__item--prev">
+                <a
+                  class="moj-pagination__link"
+                  [class.flip]="!filter.isCollapsed"
+                  href="javascript:void(0)"
+                  (click)="filter.isCollapsed = !filter.isCollapsed"
+                  ><span class="govuk-visually-hidden"> Expand {{ filter.displayName }}</span></a
+                >
+              </div>
+              <label [for]="'search-filter-' + index">{{ filter.displayName }}</label>
+            </legend>
+            @if (filter.isCollapsed) {
```

Comment:
I think isCollapsed is named incorrectly or needs to be set appropriately as currently it shows those items when expanded not collapsed